docs/source/community/host-guide.md

```diff
@@ -0,0 +1,126 @@
+*from the Jupyter Security group if needed*)
+- [ ] Make sure you have a stable internet
+connection, a good quality microphone, and a quiet environment.
+- [ ] Check the [meeting agenda](insert agenda)
+  - *The agenda is a collaboratively created document made by the participants.*
+    *Ensure a section for today's meeting is created, with today's date and a table*
+    *for participants to record their names (which can be copied/pasted from lower*
+    *entries). The agenda will also be used as meeting minutes.*
+
+&#x1F517; *Links to copy/paste into the meeting chat:*
+
+- Weekly Meeting Notes: [Insert your team’s notes here]
+- Jupyter Code of Conduct: https://jupyter.org/governance/conduct/code_of_conduct.html
+- [Subproject Name]Team Compass: 
+
+### &#x1F399; Start the meeting! 
+
+```
+Hello and welcome to our [day, month, year] Jupyter [TOPIC_HERE] call. I’m
+[host name] and I’ll be your host today.
+This is a place to for contributors to connect with each other
+and the community about [TOPIC_HERE] topics. A special highlight and welcome to all first time participants! We want all newcomers to feel
+welcome, we invite you to join in on discussions, introduce yourself,
+or add items to the agenda.
+
+Please keep in mind that this call will be recorded and posted to YouTube for the community to view. This call is a part of the Jupyter community, therefore we follow Jupyter Code of Conduct, which you can read about at jupyter.org/conduct[Meeting host insert link in the meeting minutes template]
+
+Before we begin, I want to share how topics make it on to the agenda. Presenters have submitted what they want to discuss or share ahead of time. We’ll have a little time for discussion with each one.
+```
+
+[Prepare to start the recording]
+```
+Before we start the recording, does anyone have anything they'd like
+to say off the record? (Say "We will now begin record/The recording has ended" before starting/stopping
+recording :)
+```
+
+### &#x1F3BA; Closing message 
+
+```
+Thank you to everyone for joining! I’m happy to have you all as a part
+of the Jupyter community.
+If you have any feedback, please direct it at [platform/link - could be Discourse, SSC rep email, Subproject email, Team Compass]. We’d love to hear what
+you think about the call or how to improve future meetings.
+Our next call will be [day, month]. Check out the [Jupyter community calendar](https://jupyter.org/community#calendar)to see what other types of meetings are happening in between.
+```
+
+For Community Call, delete if not: 
+```
+If you have any interest in sharing in future TOPIC_HERE  calls, please do! It doesn’t have to be a big share, a polished share, or a technical share, just show off what you do with Jupyter that excites you. Submit topics here [platform/link].
+If you’re interested in hosting a TOPIC_HERE  call, feel free to reach out. It would be great to get different people interacting with the community.
+```
```

Comment:
```suggestion
The following message is recommended for Community Calls or similar events that engage with the broader contributor community:

> If you have any interest in sharing in future [title] calls, please do! It doesn’t have to be a big share, a polished share, or a technical share, just show off what you do with Jupyter that excites you. Submit topics here [platform/link].
> If you’re interested in hosting a [title] call, feel free to reach out. It would be great to get different people interacting with the community.
```

Copy edits, rewrites intro, blockquote styling